frontend/src/components/ThreadList/ThreadList.tsx

```diff
@@ -24,6 +27,14 @@ const ThreadList = (props: ThreadListProps) => {
 
   return (
     <S.Container>
+      {noticeThread && noticeThread.authorId && (
```

Comment:
```suggestion
      {noticeThread && noticeThread.Id && (
```
공지가 없는 경우를 한번 더 확인하는거면 thread 고유번호인 Id를 검사하는게 더 좋을 것 같아요

Reply:
OK!